lib/widgets/items/item_tile_data.dart

```diff
@@ -89,6 +89,8 @@ class ItemTileData extends HookConsumerWidget {
     // upvote state until after the call has actually finished.
     final StateController<bool?> delayedUpvotedController =
         ref.watch(_delayedUpvoteStateProvider(item.id).state);
+    final bool useGestures =
+        ref.watch(useGesturesProvider).asData?.value ?? true;
```

Comment:
Similar to the page transitions instance, I prefer not passing this as a parameter. In this case, we don't even have to pass `ref` because we can use `Slidable`'s `build`'s existing `ref` to `watch` this instead.

Reply:
I didn't want to use the ref inside `Slidable` if a user on fresh install of the app sees the option in the settings and turns it off, he wouldn't be able to dismiss the walkthrough slidable..

Reply:
Oh, that makes a lot of sense, I had not considered that. Never mind then.